etl/steps/data/garden/met_office_hadley_centre/2025-01-21/near_surface_temperature.meta.yml

```diff
@@ -1,12 +1,22 @@
 definitions:
   common:
     description_key:
-      - Temperature anomalies are given in degrees Celsius relative to the average temperature over the period 1961-1990.
-      - Temperature anomalies are available for the Northern Hemisphere and the Southern Hemisphere.
-      - The global mean is calculated by averaging anomalies for northern and southern hemispheres.
+      - Temperature anomalies show how many degrees Celsius temperatures have changed compared to the 1861-1890 period. According to the source, the 1861–1890 period is often used to highlight how temperatures have changed since pre-industrial times.
+      - This early period is used as a baseline because it represents climate conditions before major human impacts.
+      - The data includes separate measurements for the Northern and Southern Hemispheres, which helps researchers analyze regional differences.
+      - The global temperature anomaly is the average of both hemisphere measurements.
+    description_from_producer: |-
+       The 1961-90 period is most often used as a baseline because it is the period recommended by the World Meteorological Organisation. In some cases other periods are used. For global average temperatures, an 1861-1890 period is sometimes used to show the warming since the "pre-industrial" period.
+    description_processing: |-
+      We switch from using 1961-1990 to using 1861-1890 as our baseline to better show how temperatures have changed since pre-industrial times. For each region, we calculate the mean temperature anomaly for 1961–1990 and for 1861–1890. The difference between these two means serves as the adjustment factor. This factor is applied uniformly to both the temperature anomalies and the confidence intervals to ensure that both the central values and the associated uncertainty bounds are correctly shifted relative to the new 1861–1890 baseline.
+    processing_level: major
+    display:
+      numDecimalPlaces: 1
   presentation:
     topic_tags:
     - Climate Change
+    grapher_config:
+      note: The period between 1861–1890 is used as a baseline to better reflect temperature changes since pre-industrial times, [as recommended by the source](https://www.metoffice.gov.uk/hadobs/indicators/index.html#:~:text=The%201961%2D90%20period%20is,other%20parts%20of%20the%20world.).
```

Comment:
I personally think that this footnote is appropriate, and enough to convey the modification. I'd say the "Adapted from" is not necessary.